howto/consume-topic/README.md

```diff
@@ -34,7 +34,7 @@ To deploy this into a Kubernetes cluster, fill in the `metadata` connection deta
 To subscribe to topics, start a web server in the programming language of your choice and listen on the following `GET` endpoint: `/dapr/subscribe`.
 The Dapr instance will call into your app, and expect a JSON value of an array of topics.
 
-*Note: The following example is written in node, but can be in any programming language*
+*Note: The following example is written in node (express.js < 4.16), but can be in any programming language*
```

Comment:
Can we just update it for >4.16, it would simplify it.

Reply:
Agree. Lets have the code we show be the latest and greatest version of express.

Reply:
The wording in this phrase is confusing (IMO). Just add a note on the express version below and it should be enough.

I agree that we should track the work to upgrade all node examples to the latest version of express.